addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -329,9 +355,7 @@ QUnit.module('web_editor', {}, function () {
                 res_id: 1,
                 mockRPC: function (route, args) {
                     if (args.model === 'ir.attachment') {
-                        if (args.method === "generate_access_token") {
-                            return Promise.resolve();
-                        }
+                            return Promise.resolve(self.data["ir.attachment"].records[0]);
```

Comment:
Try using `this` with `(route, args) => {...}` instead.

Reply:
I still don't understand this change 😕 
Instead of `return Promise.resolve(self.data["ir.attachment"].records[0]);`
Write `this(route, args) => { return Promise.resolve(self.data["ir.attachment"].records[0]); }` ?